conf/test.config

```diff
@@ -58,10 +58,5 @@ params {
     vep_cache            = "https://raw.githubusercontent.com/nf-core/test-datasets/raredisease/reference/vep_cache_and_plugins.tar.gz"
     vep_filters          = "https://raw.githubusercontent.com/nf-core/test-datasets/raredisease/reference/hgnc.txt"
     vep_cache_version    = 107
```

Comment:
needed or update to version 110?

Reply:
Test data is from 107 that's why I haven't changed it here :)